exercise7.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def calculate_room_measurement_feet(length, width):
+    """Calculate the size of a room in feet"""
+
+    area = length * width
+    return area
+
+def convert_feet_to_meters(dimensions, FEET_TO_METERS_CONVERSION):
```

Comment:
`FEET_TO_METERS_CONVERSION` is a constant and is only used in this file. Thus, you can just define it near the top of the file, say line 8, and then directly use it in your function. The constant will be available to the function. Remember that `FEET_TO_METERS_CONVERSION` is just the name of the function's parameter, it is not the _parameter_ that is eventually passed in. 

This function should look like:

```
def convert_feet_to_meters(dimensions):
  ...
```